versions/3.1.0.md

```diff
@@ -2241,10 +2243,18 @@ description: Pets operations
 
 A simple object to allow referencing other components in the OpenAPI document, internally and externally.
 
+The `$ref` string value contains a URI [RFC3986](https://tools.ietf.org/html/rfc3986), which identifies the location of the value being referenced.
+
+Resolution of a Reference Object SHOULD yield the referenced value. Implementations MAY choose to replace the reference with the referenced value.
```

Comment:
`Implementations MAY choose to replace the reference with the referenced value.` - why mention this?

Reply:
Because it is wording directly from https://tools.ietf.org/html/draft-pbryan-zyp-json-ref-03 so represents the status quo of what we were referencing in 3.0.x

Reply:
If a concern is that replacing the Reference Object with the referenced value is in conflict with the notion of having sibling properties to the `$ref`, then we can pinch wording from JSON Schema such as:

> Its results are the results of the referenced ~~schema~~ value.